pkg/network/node/networkpolicy.go

```diff
@@ -794,6 +798,35 @@ func (np *networkPolicyPlugin) parsePeerFlows(npns *npNamespace, npp *npPolicy,
 				// checking the source VNID...
 				npp.watchesNamespaces = true
 				peerFlows = append(peerFlows, np.selectNamespaces(peer.NamespaceSelector, dir)...)
+				if _, ok := os.LookupEnv(migrtionEnvVar); ok {
+					// In SDN live migration mode, we can't use source VNID to
+					// distinguish namespaces for traffic from OVN nodes.
+					// So instead we pretend the rule was a combined
+					// namespaceSelector+podSelector rule with a match-all
+					// podSelector, and generate per-pod-IP match rules.
+					npp.watchesAllPods = true
+					peerFlows = append(peerFlows, np.selectPodsFromNamespaces(peer.NamespaceSelector, &metav1.LabelSelector{}, dir)...)
+
+					// If the host network namespace is selected, Add rules for
+					// the OVN mp0 IP of each node.
```

Comment:
SDN treats all non-pod-cidr traffic as "host network", so we just need to accept anything that's not in `npp.node.clusterCIDRs`.

The "not" makes this difficult though... the easiest way to deal with this would probably be to update [this rule](https://github.com/openshift/sdn/blob/d515d15883394318af2c55e75e17291a65d023b2/pkg/network/node/ovscontroller.go#L258) to add `load:1->NXM_NX_REG3[]` to the actions, to tag all nw_src=clusterCIDR with reg3=1 (doesn't need to be conditional on migration). Then here you can just check `reg3=0` here to match everything that didn't match that (ie, non-clusterCIDR-source traffic).

Oh! And either way, you need to bump `ruleVersion` in ovscontroller.go (which will require updates to the unit tests as well).

Reply:
But the IP of the mp0 interface is in the range of the clusterCIDR. It's the second IP of the node subnet.

Reply:
I added unit test and bumped the `ruleVersion`. PTAL